source/documentation/fuseki2/fuseki-modules.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Fuseki Modules
+---
+
+<em>Fuseki modules</em> are a mechniam to include extension code into a Fuseki
+server. Modules are invoked during the process of building a [Fuseki
+Main](./fuseki-main.html) server. The module can modify the server
+configuration, add new functiomnality, or react to a server being built and
+started.
+
```

Comment:
As a brand new feature may want an explicit note about the version when support was added for this

Reply:
Done.

(Personal note - I don't mind such notes even if it does hint at "support" for older versions. We have quite good at compatibility so upgrading to a recent version is better. Even better, if we could run support branches and releases. "**_IF_**" ...